contracts/incentives/FarmFactory.sol

```diff
@@ -122,19 +119,6 @@ contract FarmFactory is CloneFactory {
 			address(avatar)
 		);
 
-		//call notify reward amount
-		(success,) = controller.genericCall(
```

Comment:
Why are we removing this call and require statement?

Reply:
I moved this functionality to `initialization`  function in `StakingRewards` contract